lib/diagnostic.js

```diff
@@ -17,7 +17,16 @@
 // Methods: sell, call
 
 // Answer Starts Here
-//
+/*
+Entity: Calender
+Attributes: day, month, year, time
```

Comment:
Can you be more specific about what `day`,`month`,`year`, and `time` are? What do they refer to?